ftw/book/tests/test_uninstall.py

```diff
@@ -16,8 +16,4 @@ class TestGenericSetupUninstall(TestCase, GenericSetupUninstallMixin):
 
     @property
     def skip_files(self):
-        if getFSVersionTuple() < (4, 3):
-            # Plone <= 4.2 does not support removing thins in tinymce.xml
-            return ('tinymce.xml')
-        else:
-            return ()
+        return ('tinymce.xml')
```

Comment:
We should return the value from the `else` branch for Plone 4.3. 
For Plone < 4.3 (i.e. 4.2, 4.1, etc) we used to return the value from the `if`.

Reply:
So actually nothing at all? That is looks like:?
```
def skip_files(self):
    return ()
```

Reply:
Good point. I think we could remove the property altogether then.